packaging/suse/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: trento/trento-wanda:latest
+#!BuildTag: trento/trento-wanda:%%VERSION%%
+#!BuildTag: trento/trento-wanda:%%VERSION%%-build%RELEASE%
+#!UseOBSRepositories
+
+FROM bci/rust:1.66 AS release
```

Comment:
Just out of curiosity, why do we need a `rust` image here if the rust files are already vendored?

Reply:
Because the deps are vendored ineed, not built. They get built later in OBS.